protocol/patricia/patricia.go

```diff
@@ -60,29 +60,20 @@ func walk(n *node, walkFn WalkFunc) error {
 	return err
 }
 
-// ContainsKey returns true if the key contains the provided
-// key, without checking its corresponding hash.
-func (t *Tree) ContainsKey(bkey []byte) bool {
-	if t.root == nil {
-		return false
-	}
-	return lookup(t.root, bitKey(bkey)) != nil
-}
-
-// Contains returns true if the tree contains the provided
-// key, value pair.
-func (t *Tree) Contains(bkey, val []byte) bool {
+// Contains returns whether the tree contains (item, item)
+// as a (key, value) pair.
+func (t *Tree) Contains(item []byte) bool {
 	if t.root == nil {
 		return false
 	}
 
-	key := bitKey(bkey)
+	key := bitKey(item)
 	n := lookup(t.root, key)
 
 	var hash bc.Hash
 	h := sha3pool.Get256()
 	h.Write(leafPrefix)
-	h.Write(val[:])
+	h.Write(item)
```

Comment:
I don't want to leave out the value comparison just yet because this package doesn't enforce that keys and values are equal. Once we've updated Insert, then I think that would be safe, right? Also, long run, it should just be "Contains" because it's semantically a set -- there are no keys or values, only items in the set.